lib/ohai/plugins/solaris2/network.rb

```diff
@@ -104,7 +104,8 @@ def full_interface_name(iface, part_name, index)
 
     so.stdout.lines do |line|
       # regex: http://rubular.com/r/Iag7JLVTVe
```

Comment:
Can you update the rubular perma link to include the new values you're trying to match?

Reply:
I added an additional rubular link as the old link is still relevant in explaining the conditional